pkg/client/clientset_generated/internalclientset/typed/policy/internalversion/eviction_expansion.go

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package internalversion
+
+import (
+	"strings"
+
+	policy "k8s.io/kubernetes/pkg/apis/policy"
+)
+
+// The EvictionExpansion interface allows manually adding extra methods to the ScaleInterface.
+type EvictionExpansion interface {
+	Evict(eviction *policy.Eviction) error
+}
+
+func (c *evictions) Evict(eviction *policy.Eviction) error {
```

Comment:
Don't hack the pkg/restclient.
Could you get the c.Client.(*restclient.RESTClient), modify its [versionedAPIPath](https://github.com/kubernetes/kubernetes/blob/master/pkg/client/restclient/client.go#L65), use the modified RESTClient to send request?